packages/api/src/services/library_item.ts

```diff
@@ -238,10 +239,16 @@ const buildWhereClause = (
     args.matchFilters.forEach((filter) => {
       const param = `match_${filter.field}`
       queryBuilder.andWhere(
-        `websearch_to_tsquery('english', :${param}) @@ library_item.${filter.field}_tsv`,
-        {
-          [param]: filter.value,
-        }
+        new Brackets((qb) => {
+          qb.andWhere(
+            `websearch_to_tsquery('english', :${param}) @@ library_item.${filter.field}_tsv`,
+            {
+              [param]: filter.value,
+            }
+          ).orWhere(`${filter.field} ILIKE :value`, {
```

Comment:
probably best to add alias here in case typeorm is stupid

Reply:
Oh i had a little trouble with this because fieldname is like `site_name` I couldn't do:

orWhere({
   site_name: value
})

because typeORM expects `siteName: value`

Reply:
yeah, we'd best use raw sql here like `library_item.${filter.field} ILIKE :value` because typeorm is stupid and the update query for bulk action would fail if we do object literal 😢